### ISSUE TITLE
Example Photos Not Shown

### ISSUE THREAD

**Issue by jeanettepayne** [learn-co-curriculum/positioning-elements-with-css#7]
It looks like there are possible missing images in the lesson. It will say "see example below" then reference the photo, but I'm not being shown any images.

**Comment by Sdcrouse** [comment]
Yeah, as far as I can tell, it's a weird issue with `<iframe>`s and either Markdown or the Learn app itself. I don't know how to fix it (I'm a student as well). I did find a temporary fix, though: 

1. Go to the lesson's README at https://github.com/learn-co-curriculum/positioning-elements-with-css/blob/master/README.md
2. In the top-right corner, click the "Raw" button. You'll be able to see the raw Markdown for this lesson.
3. Each time you see an `<iframe>`, copy it and go back to the Learn lesson.
4. Right-click on the paragraph right above where that `<iframe>` should be and select "Inspect".
5. The `<p>` tag for that paragraph should be selected. Now, right-click on it and select "Edit as HTML".
6. Next to the closing `<p>` tag, paste in the `<iframe>`.
7. Click just to the left of that `<p>` tag without going outside of the Inspector. The `<iframe>` should now show up on the page properly.
8. Lastly, repeat Steps 3-7 for the other `<iframe>` elements.

Hope that helps!
